README.md

```diff
@@ -54,6 +54,17 @@ Please, set valid commit message, to increment the right version
 ### To increase version (X.y.z) version
 `````perf: your commit message...`````
 
+# Logos
+
+- all logos should be in SVG so mobile apps wouldn't need to support 5 images of each easset that will grealy increase the app size and complexity on frontend.
```

Comment:
- All logos must be in SVG format to avoid the need for mobile apps to support multiple image assets, which would significantly increase app size and frontend complexity
- All logos should have a fixed size of 36x36